build/src/javascript/view/columnView.js

```diff
@@ -0,0 +1,211 @@
+import { putUpdatedCardData } from "../model/cardModel.js";
+import { debounce, moveAt } from "../util/util.js";
+import {
+  renderRegisterCard,
+  addRegisterBtnsListener,
+  addCardDoubleClickEvent,
+  changeOriginalCardStyle,
+  getUpdatedCardContent,
+} from "./cardView.js";
+
+let [isDraggable, dragStart, isDowned] = [false, false, false];
+let [shiftX, shiftY] = [0, 0];
+let $originalCard;
+
+export function renderColumn(columnId, todos) {
+  const columnSelector = `#${columnId}`;
+  const $column = document.querySelector(columnSelector);
```

Comment:
이정도는 합쳐서 써줘도 될 것 같습니다. :)
```suggestion
  const $column = document.querySelector(`#${columnId}`);
```